java-source/build.gradle

```diff
@@ -38,7 +38,7 @@ dependencies {
     compile "net.corda:node:$corda_version"
     compile "net.corda:corda:$corda_version"
     compile "net.corda:test-utils:$corda_version"
-    runtime "net.corda:corda-webserver:$corda_version"
+    runtime "net.corda:webserver:$corda_version"
```

Comment:
This should be;

    compile "net.corda:webserver:$corda_version"
    runtime "net.corda:corda-webserver:$corda_version"

Both are needed.